### PR TITLE
Adding Cardano Foundation Explorer and CExplorer to showcases

### PR DESCRIPTION
**Pull request by Kammerlo**
👋 Hello there! Welcome. Please follow the steps below to tell us about your contribution.

1. Please complete a Checklist
2. Fill in all sections of the template
3. Click "Create pull request"

## Checklist

- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have read the [Showcase Requirements](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/showcases.js)
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 

## Showcase addition

* Title: Cardano Foundation Explorer
  * Description: A Cardano explorer aiming to serve the needs of a non technical target audience like regulators and policy makers built by Cardano Foundation, currently under development
  * Website: https://beta.explorer.cardano.org
  * Source: https://github.com/cardano-foundation/cf-explorer
  * Tags:
    * `explorer`

* Title: CExplorer
  * Description: An independent Cardano Explorer that provides useful all-in-one dashboards additionally.
  * Website: https://cexplorer.io/
  * Tags:
    * `explorer`

### CODE REVIEW

**Review by katomm — APPROVED** [review]
LGTM

**Review by rphair — COMMENTED** [review]
@Kammerlo `cexplorer.io` is already listed as a builder tool (`/src/data/builder-tools.js`):
```
  {
    title: "Cexplorer",
    description: "Rich featured blockchain explorer with support of all testnet instances and an API.",
    preview: require("./builder-tools/cexplorer.png"),
    website: "https://cexplorer.io",
    getstarted: null,
    tags: ["operatortool"],
  },  
```
We've had a long standing policy of only listing each tool once even if eligible for both portfolios.  There are a few PRs you can see in the history of the times Adapools / Cexplorer has been added, removed, and modified.  The discussion of which portfolio (Showcase or Builder Tools) was in the most recent PR the as I recall... I'll find & link to it again if any difficulty finding it.

@katomm do we still have the same policy?  In my opinion some tools are so universally used that it makes sense for their 2 capacities to show in the 2 portfolios (in this case, both an operator tool and end-user analytics).

**Comment by katomm** [comment]
Great catch @rphair. I should have provided more context, but was on the phone when I reviewed this entry. I discussed this PR with @Kammerlo upfront and we both were looking for CExplorer is listed already in the explorer category (and were unable to find it) because we want to link from another page to all the explorers.

I remember the back and forth with Adapools and CExplorer and want to make the proposal here to add CExplorer to showcase in the category "explorers" and remove it from Builder Tools. I understand they have APIs but it is still an explorer. 

(happy to make the pull request and sorry for the confusion)

File: src/data/showcases.js [hunk 1]
```diff
@@ -268,11 +268,20 @@ export const Showcases = [
...
   {
-    title: "Cardano Explorer",
+    title: "Cardano Foundation Explorer",
     description:
-      "The Cardano explorer built by IOHK, one of the founding entities of Cardano.",
-    preview: require("./showcase/cardanoexplorer.png"),
-    website: "https://explorer.cardano.org",
+      "A Cardano explorer focused on the needs of a non technical target audience like regulators and policy makers built by Cardano Foundation, currently under development",
+    preview: require("./showcase/cf-explorer.png"),
+    website: "https://beta.explorer.cardano.org",
```
Comment (rdlrt) [inline src/data/showcases.js:275]: what are the plans/path to move from beta.explorer.cardano.org to explorer.cardano.org?

**Review by rdlrt — APPROVED** [review]

File: src/data/showcases.js (hunk 1 above)
Comment (katomm) [reply src/data/showcases.js:275]: We are figuring this out as we speak. What is clear already is that what is currently running on explorer.cardano.org will not make it to 9.1. 

Options on the table atm: 
- Replace it
- Landing page on explorer.cardano.org who lists other explorers. (That someone needs to maintain)
- Redirect to showcase with the explorer category. 

Any opinions welcome.

Comment (rdlrt) [reply src/data/showcases.js:275]: Replace it =] 
A landing page for explorers could be with plural 😉

**Review by rphair — APPROVED** [review]

**Comment by katomm** [comment]
FYI: I have opened the PR to remove the cexplorer from builder tools.

File: src/data/showcases.js (hunk 1 above)
Comment (katomm) [reply src/data/showcases.js:275]: You are not wrong sir.

**Pull request merged**